app/models/project/Project.scala

```diff
@@ -504,6 +504,34 @@ case class Project(override val id: Option[Int] = None,
     if (isDefined) update(RecommendedVersionId) else Future.unit
   }
 
+  /**
+    * Get a collection of tags that represent a project through its versions
+    */
+  def tags(implicit ec: ExecutionContext, service: ModelService = null): Future[List[Tag]] = {
```

Comment:
Can we please try to get away from this pattern with an implicit value with a default null?

Reply:
I just copied the code from `Version@tags`